common/src/main/java/com/viaversion/viaversion/protocols/protocol1_13to1_12_2/storage/BlockConnectionStorage.java

```diff
@@ -85,7 +89,7 @@ public int get(int x, int y, int z) {
 
     public void remove(int x, int y, int z) {
```

Comment:
Also need to set the cached object to null at the end of the this method where a chunk entry is removed if empty